app/config/urls.py

```diff
@@ -30,4 +30,5 @@
     urlpatterns = [
         path('__debug__/', include(debug_toolbar.urls)),
         path('tests/test_http_500', TestHttp500.as_view()),
+        path('prometheus/', include('django_prometheus.urls')),
```

Comment:
Is it on purpose that you add the prometheus endpoint only for debug ? Just as a side note, I've found out that our docker image build with --target debug suffer  from  a memory leak ! The --target production is not touched by this memory leak.

Reply:
No, of course it should not only be in debug, thanks.